services/chronograf/values-squash-sandbox.yaml

```diff
@@ -0,0 +1,88 @@
+chronograf:
+  ## Image Settings
+  ##
+  image:
+    repository: "chronograf"
+    tag: 1.8.6
+    pullPolicy: IfNotPresent
+
+  ## Specify a service type
+  ## ClusterIP is default
+  ## ref: http://kubernetes.io/docs/user-guide/services/
+  ##
+  service:
+    replicas: 1
+    type: ClusterIP
+
+  ## Persist data to a persistent volume
+  ##
+  persistence:
+    enabled: true
+    ## chronograf data Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    ##
+    # storageClass: "-"
+    accessMode: ReadWriteOnce
+    size: 16Gi
+
+  ## Configure resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
+    requests:
+      memory: 256Mi
+      cpu: 0.1
+    limits:
+      memory: 2Gi
+      cpu: 2
+
+  ## Node labels for pod assignment
+  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+  #
+  nodeSelector: {}
+
+  ## Tolerations for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+
+  ## Affinity for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {}
+
+  ## Configure the ingress object to hook into existing infastructure
+  ## ref : http://kubernetes.io/docs/user-guide/ingress/
+  ## OPTIONALLY you can set .Values.ingress.secretName to set which secret to use
+  ##
+  ingress:
+    enabled: true
+    tls: true
+    secretName: tls-certs
```

Comment:
I think you need the cluster-issuer configuration here as well.

In practice, this is a little odd since multiple ingresses that share the same hostname can't have separate TLS certificates, so one of these is going to win and the rest will lose. What we have currently done for lsp-deploy is make nublado win and not configure TLS for anything else. You don't have nublado in this cluster so that doesn't work. I think this works since all ingresses that define TLS obtain essentially identical certs, but be aware that one of them will win (based on age of the resource) and the rest will be ignored.

Reply:
Good catch, let me review that I think I can set `tls: false` since I'm using path base routing now.

Reply:
That's right, I can use`tls: false` here.

Reply:
I'm amazed. I've disable tls for `influxdb` and the `squash-api` as well. I don't need the `tls-certs` secret in each app namespace anymore, things are much simpler with path base routing.

Reply:
Ok, too early to say that. I just confirmed what you said. I need at least one service creating the `tls-certs` even if it is created on the app namespace. Since all services share the same base hostname this works. So I think I'll choose to configure tls in the `squash-api`.